solutions/observability/observability-ai-assistant.md

```diff
@@ -130,12 +130,25 @@ To add external data to the knowledge base in {{kib}}:
 
 [Search connectors](elasticsearch://reference/search-connectors/index.md) index content from external sources like GitHub, Confluence, Google Drive, Jira, S3, Teams, and Slack to improve the AI Assistant's responses.
 
-**Requirements and limitations:**
+#### Requirements and limitations
+
 - For stack 9.0.0+ or {{serverless-short}}, connectors must be [self-managed](elasticsearch://reference/search-connectors/self-managed-connectors.md)
```

Comment:
Optional edit - just aligning it with punctuation in [this other reqs section](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/1770/solutions/observability/observability-ai-assistant#obs-ai-requirements).
```suggestion
- For {{stack}} 9.0.0+ or {{serverless-short}}, connectors must be [self-managed](elasticsearch://reference/search-connectors/self-managed-connectors.md).
```